Prose/ProseLib/Sources/ConversationFeature/Chat/Chat.swift

```diff
@@ -126,7 +126,7 @@ struct ChatView: NSViewRepresentable {
     webView.loadFileURL(Files.messagingHtml.url, allowingReadAccessTo: Files.messagingHtml.url)
 
     context.coordinator.ffi = FFI { [weak webView] jsString, completion in
-      jsLogger.trace("\(jsString)")
+      jsLogger.trace("\(jsString, privacy: .public)")
```

Comment:
Erm… are you sure about this? Logs contain unencrypted messages, I'd prefer not to leak them here…

Reply:
Good point. However `trace` per default obscures the output. I don't know why it is so hard to find documentation on that (it could be explained [here](https://developer.apple.com/documentation/os/logger), but isn't) but I believe this is ok and well:


From the 2020 WWDC Talk ("Explore Logging in Swift"):

> Debug level messages are not persisted, which means they cannot be retrieved after the app has completed execution. Info error messages are mostly not persisted, except when they are generated a few moments before a log collect command. Messages logged at every other level are persisted, and you can retrieve them later. 

> Logging at the debug level is so fast because debug messages are not persisted at all. They are discarded when the logs aren't being streamed. Further, the Swift compiler uses sophisticated optimizations to ensure that the code that creates the messages is not even executed when the debug messages are discarded. This means that you can log verbose messages at the debug level and call expensive functions to construct messages. Your users won't pay the cost for them.

I have no idea why `trace` isn't even mentioned however the docs say this:
> This method is functionally equivilent to the debug(_:) method.

More info: https://www.wwdcnotes.com/notes/wwdc20/10168/

---

Turns out setting `OS_ACTIVITY_DT_MODE` to `NO` redacts messages even during debugging which is why I wasn't seeing anything.

Reply:
> I have no idea why trace isn't even mentioned however the docs say this:

`.trace` is a function, not a log level properly speaking. [OSLogType](https://developer.apple.com/documentation/os/oslogtype)s are log levels, and we can see there that `.trace` doesn't exist. `Logger.trace(_:)` just writes a `.debug` log but obscures it I believe. Which now makes me wonder why I use it 🤔

Reply:
> which is why I wasn't seeing anything

Do you mean it's not obfuscated then?